### PR TITLE
Fix parsing bug in 3d rising bubble driver.

### PR DESCRIPTION
**Pull request by sriharshakandala**
- [x] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [x] Unit tests are included OR N/A.
- [x] Code is exercised in an integration test OR N/A.
- [x] Documentation has been added/updated OR N/A.

### CODE REVIEW

**Comment by sriharshakandala** [comment]
bors try

File: examples/hybrid/box/bubble_3d_invariant_rhoe.jl [hunk 1]
```diff
@@ -358,13 +358,13 @@ function bubble_3d_invariant_ρe(ARGS, comms_ctx, ::Type{FT}) where {FT}
...
     if resolution == "custom"
         args = (
-            FT(get(ARGS, 3, 1000)),
-            FT(get(ARGS, 4, 1000)),
-            get(ARGS, 5, 4),
-            get(ARGS, 6, 16),
-            get(ARGS, 7, 3),
-            FT(get(ARGS, 8, 0.05)),
-            FT(get(ARGS, 9, 700.0)),
+            parse(FT, get(ARGS, 3, 1000)),
```
Comment (simonbyrne) [inline examples/hybrid/box/bubble_3d_invariant_rhoe.jl:361]:
```suggestion
            parse(FT, get(ARGS, 3, "1000")),
```
and same below

**Comment by sriharshakandala** [comment]
bors try

**Comment by sriharshakandala** [comment]
bors try-

**Comment by sriharshakandala** [comment]
bors try

**Review by simonbyrne — APPROVED** [review]

**Pull request merged**